README.md

```diff
@@ -0,0 +1,48 @@
+## Getting Started
+
+Refer to the [Yesod Quickstart guide](https://www.yesodweb.com/page/quickstart) for install instructions.
+
+## Database Setup
+
+After installing Postgres, run:
+
+```
+createuser PROJECTNAME_LOWER --password PROJECTNAME --superuser
+createdb PROJECTNAME_LOWER
+createdb PROJECTNAME_LOWER_test
+```
+
+## Development
+
+Start a development server with:
+
+```
+stack exec -- yesod devel
```

Comment:
Should probably command on needing to run `stack build yesod-bin` first.

Reply:
So that's covered in the Quick Start guide, but I think it would be good to pull the stuff from the QuickStart guide into this README (the quickstart guide hasn't changed that much since the transition to Stack).

The Quickstart guide recommends this command:

```
stack build yesod-bin cabal-install --install-ghc
```

Do you know offhand if `cabal-install` is actually necessary?

Reply:
`cabal-install` is no longer necessary, we can remove that recommendation.

Reply:
Nice, saw you removed it from the quickstart guide as well 👍